ArchiSteamFarm/Localization/Strings.zh-CN.resx

```diff
@@ -1,17 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
You don't modify culture-specific files, as per the `README.md` in the `Localization` folder. Remove this file from PR.